src/app/core/services/http/http-file-download.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core'
+
+export interface FileDownloaderService {
+  downloadText(text: string, filename: string): void
+}
+
+@Injectable()
+export class HttpFileDownloadService implements FileDownloaderService {
+  public downloadText(text: string, filename: string): void {
+    const anchorElement: HTMLAnchorElement = document.createElement('a')
+    anchorElement.setAttribute('href', 'data:text/plain;charset=utf-u,' + encodeURIComponent(text))
```

Comment:
Both standards works fine, but is better to be utf-8, Done.

Reply:
Can you enlighten me on what `UTF-U` is? :D I haven't been able to find something that describes it.

Reply:
I think was somehow Unicode convention, but I never check it and now I don't see any more information too. I was generally going to say that logic works both ways (with utf-u & utf-8), but utf-8 encoding is better because I know exactly how it works. Most likely html detect invalid format (utf-u) and automatically change it to utf-8.